### ISSUE TITLE
Question about mask accuracy calculation

### ISSUE THREAD

**Issue by jwubz123** [facebookresearch/meshrcnn#115]
In mask accurate calculation, we only have pred_mask_logits without sigmoid, the value is out of [0,1], (pred_mask_logits > 0.5) selection seems to be meaningless, is there any processing about pred_mask_logits that I overlooked?  [  mask_accurate = (pred_mask_logits > 0.5) == (gt_masks > 0.5)](https://github.com/facebookresearch/meshrcnn/blob/b2e8e5898b415b4fb7f12ac725afc7e2268ffb03/meshrcnn/modeling/roi_heads/mask_head.py#L63)
